html/semantics/embedded-content/the-img-element/update-the-image-data/current-request-microtask.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<title>An img's current request should be updated in a microtask after selecting an image source</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+
+<script>
+const picture = document.createElement("picture");
```

Comment:
This setup all looks very safe, but if it could plausibly fail then wrapping it in `setup()` would be best.

But since there's just a single test here I'd suggest simplifying with `single_test()`, also avoiding some callback wrapping.

Reply:
The opt-in to single test is `setup({ single_test: true });`

Alternatively move up the `async_test` line.